src/2_session.js

```diff
@@ -50,6 +50,31 @@ session.update = function(storage, newData) {
 	storage.set('branch_session', data);
 };
 
+/**
+ * Patches a field in localStorage or sessionStorage or both. 
+ * @param {storage} storage
+ * @param {Object} data
+ * @param {boolean=} druable
+ */
+session.patch = function(storage, data, druable){
+
+	const merge = (source, patch) =>{
+		return utils.merge(source, patch);
+	}
+
+	const session = storage.get('branch_session', false) || {}
+	if(session){
+		storage.set('branch_session', goog.json.serialize(merge(session, data)));		
+	}
+
+	if(druable){
```

Comment:
Can we change this variable name to updateLocalStorage?

Reply:
👍